### PR TITLE
Fix grammar in SKIP 1

### LINKED ISSUE #7075: Link text incorrect in SKIP 1
### Description:

> Changes to this governance model or our mission, vision, and values require a [Improvement proposals (SKIPs)](https://scikit-image.org/docs/stable/skips/1-governance.html#skip) and follow the decision-making process outlined above, unless there is unanimous agreement from core developers on the change.

Does not parse, should be "require an improvement proposal (SKIP)"

### Way to reproduce:

_No response_

### Version information:

_No response_

### PR DESCRIPTION
**Pull request by decorouz**
## Description
Fix incorrect text in SKIP 1, closes #7075.

## Checklist

- A descriptive but concise pull request title
- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [Unit tests](https://scikit-image.org/docs/dev/development/contribute.html#testing)
- A gallery example in `./doc/examples` for new features
- [Contribution guide](https://scikit-image.org/docs/dev/development/contribute.html) is followed

## Release note

Summarize the introduced changes in the code block below in one or a few sentences. The
summary will be included in the next release notes automatically:

```release-note
...
```

### CODE REVIEW

File: doc/source/skips/1-governance.rst [hunk 1]
```diff
@@ -157,7 +157,7 @@ are made according to the following rules:
   decision-making process outlined above.
 
 - **Changes to this governance model or our mission, vision, and values**
-  require a :ref:`skip` and follow the decision-making process outlined above,
+  require an :ref:`skip` and follow the decision-making process outlined above,
```
Comment (stefanv) [inline doc/source/skips/1-governance.rst:160]: Thanks! It should say "require a SKIP", but the text renders as "require a ... many words here ....".

Comment (decorouz) [reply doc/source/skips/1-governance.rst:160]: It should say "requires a SKIP" you mean ?

Comment (lagru) [reply doc/source/skips/1-governance.rst]: No, the ["an" is used only if the following word is not started by a consonant sound](https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an). Here "SKIP" starts with a consonant sound, so "a" should be correct.

Also, I think "require" (not "requires") is the correct term as it references the plural "changes". The "s" is only used for singular third person.

I'm not a native speaker mind you, so my gut feeling and explanations may be totally off. :sweat_smile:

Comment (lagru) [reply doc/source/skips/1-governance.rst]: So I agree with what Stéfan suggests: 
```
require a :ref:`SKIP <skip>`
```

**Comment by lagru** [comment]
Note, I think this references SKIP 1 and not SKIP 0 (https://github.com/scikit-image/scikit-image/issues/7075 also got this wrong). I updated both PR titles accordingly. 

And I also inserted the keyword "closes" in front of this PR's description. See [Using keywords in issues and pull requests](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) if you don't know this behavior.

**Review by lagru — APPROVED** [review]
Perfect and thanks @decorouz!

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**